CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ All notable changes to this project will be documented in this file.
 ### Added
 - Example for Issue #1716 showing poor tick spacing on DateTimeAxis with interval types of Weeks or Years
 - Example for label placement on BarSeries with non-zero BaseValue (#1726)
+- ExtrapolationLineSeries and examples
```

Comment:
Should mention motivating issue (helps future maintainers find design discussion):

    - ExtrapolationLineSeries and examples (#1740)

Reply:
Done.